src/main/java/uk/gov/companieshouse/itemhandler/email/CertifiedDocument.java

```diff
@@ -0,0 +1,40 @@
+package uk.gov.companieshouse.itemhandler.email;
+
+public class CertifiedDocument {
+    private String dateFiled;
+    private String type;
+    private String description;
+    private String fee;
```

Comment:
There should also be descriptionValues and cost, if you do a pull on certified copies api they should be there. Although they might not be needed in this?

Reply:
This class only contains properties that are needed to be displayed in the email, which is why other properties have been omitted from it.